infra/base-images/base-runner/Dockerfile

```diff
@@ -94,13 +94,18 @@ RUN wget https://download.java.net/java/GA/jdk15.0.2/0d1cfde4252546c6931946de8db
 # Install Java AWT dependencies.
 RUN apt-get install libxext-dev -y
 
+# Install JaCoCo for JVM coverage.
+RUN wget https://repo1.maven.org/maven2/org/jacoco/org.jacoco.cli/0.8.7/org.jacoco.cli-0.8.7-nodeps.jar -O /opt/jacoco-cli.jar && \
```

Comment:
While the standard build process using Maven would build JaCoCo itself from source, it will still fetch and run dozens of .jar's (both JaCoCo dependencies and Maven plugins).

Could you explain the reasons for preferring from source builds for tooling in more detail? Maybe then I can come up with a better solution (e.g., Jazzer builds some parts of JaCoCo with Bazel, which does not involve fetching and running any jars).

Reply:
For supply chain security, it is preferable to build from source. Since your code is on github, can we just use bazel workflow to build it.

Reply:
>Could you explain the reasons for preferring from source builds for tooling in more detail?

Yeah. Don't like trusting arbitrary binaries so much.
Let's just download a particular version of the jars and check the shasum like so:
echo "37df187b76888101ecd745282e9cd1ad4ea508d6  org.jacoco.agent-0.8.7-runtime.jar" | shasum  --check

Reply:
I added the shasum checks.

> Since your code is on github, can you just use bazel workflow to build it.

We only build a very small (and almost dependency free) part of JaCoCo within Bazel, but here we need the full application (HTML report generation, agent, ...).